docs/src/modules/java/pages/eventsourced.adoc

```diff
@@ -134,48 +134,49 @@ By default, the generated constructor has an `EventSourcedEntityContext` paramet
-<2> This constructor allows us to set a state on the Entity, so we can start the test directly from the desired state instead of having to send all the commands that would produce such a state.
-<3> This method returns the current state of the Entity. 
-<4> This method returns the accumulated events from all the executed commands sent to the entity.
-<5> This method calls to the method with the same name in the Event Source Entity passed in the constructor.
+<1> creating the TestKit passing the constructor of the Entity.
+<2> calling the method `addItem` from the Entity in the `ShoppingCartTestKit`.
+<3> calling the method `addItem` from the Entity in the `ShoppingCartTestKit`.
+<4> checking the `EventSourcedResult` of the first call to `addItem`.
+<5> checking the `EventSourcedResult` of all the calls to `addItem`.
+<6> retrieving the first event generated from the first call to `addItem`.
+<7> retrieving the second event generated from the first call to `addItem`. There is no such event as our implementation only generates one event when addItem it's called.
+<8> retrieving the response from the call to `addItem`.
+<9> retrieving the state of the entity after all the calls to `addItem`.
+
 
+*EventSourcedResult*
 
-Once we call a  method on the entity this gives us back a link:{attachmentsdir}/api/com/akkaserverless/javasdk/testkit/EventSourcedResult.html[`EventSourcedResult` {tab-icon}, window="new"]. This class has methods that we can use to assert our expectations, such as:
+Calling an entity method through the TestKit gives us back an link:{attachmentsdir}/api/com/akkaserverless/javasdk/testkit/EventSourcedResult.html[`EventSourcedResult` {tab-icon}, window="new"]. This class has methods that we can use to assert our expectations, such as:
 
 * `getReply` is the response of the method called in the test
 * `getAllEvents` are the events generated upon the method called in the test. This list can be inspected multiple times. This method is idempotent.
 * `getNextEventOfType` is the first of the events generated upon the method called in the test. This event gets consumed once is inspected. This method is not idempotent.
 
-Finally, the following snippet shows how to use the autogenerated TestKit to test our implementation. 
 
-.src/test/java/com/example/shoppingcart/domain/ShoppingCartTest.java
+*ShoppingCartTestKit*
+
+This class is generated by AkkaServerless when the project is compiled. The following is a simplification showing the signatures of its available methods.
+
+.target/generated-test-sources/com/example/shoppingcart/domain/ShoppingCartTestKit.java
 [source,java]
 ----
-include::example$java-eventsourced-shopping-cart/src/test/java/com/example/shoppingcart/domain/ShoppingCartTest.java[] 
+include::example$testkit/java-eventsourced-shopping-cart/intarget/generated-test-sources/akkaserverless/com/example/shoppingcart/domain/ShoppingCartTestKit.java[] 
```

Comment:
I don't think we should show this fake ShoppingCartTestKit.java at all

Isn't this illustrated and described enough in above ShoppingCartTest.java ?

Reply:
sounds good. I'll remove this